Engineering/SourceControl.md

```diff
@@ -15,11 +15,15 @@ There are many different options when working with Source Control. In [CSE](../C
 ## General Guidance
 Consistency is crucial, therefore the team has to agree on their approach before they start to code. At a minimum, the team should do the following:
 * agree on their **branch**, **release** and **merge strategy**
+* agree on tooling for **conflict resolution** and other source control operations (by OS)
+  * VSCode extensions
+  * [CodeFlow](https://www.1eswiki.com/wiki/CodeFlow) 
```

Comment:
1eswiki.com only works for people in the Microsoft tenant, so public users won't be able to use this link. Since this is public documentation, we probably don't want to link to internal Microsoft pages.